sgp4/model.py

```diff
@@ -48,6 +48,15 @@ def twoline2rv(cls, line1, line2):
         self.epochyr %= 100  # undo my non-standard 4-digit year
         return self
 
+    @classmethod
+    def Satellite(cls):
```

Comment:
This classmethod is slightly surprising. Could you explain it?

Reply:
I can't - just following your example.